test/integration/scene_broadcaster_system.cc

```diff
@@ -162,7 +162,7 @@ TEST_P(SceneBroadcasterTest, SceneGraph)
 
   EXPECT_TRUE(node.Request("/world/default/scene/graph", timeout, res, result));
   EXPECT_TRUE(result);
-
+  std::cout << res.data() << "\n";
```

Comment:
remove?

Reply:
Done in dfb8ff66ab689529fb0574210a89a3bec797e473